src/app/components/send-coins-form/send-coins-form.component.ts

```diff
@@ -93,6 +93,25 @@ export class SendCoinsFormComponent implements OnInit, OnDestroy {
     })
   }
 
+  modelUpdated(input: string){
+    this.invalidateEstimate()
+    let removedError
+    
+    if (input === 'amount') {
+      removedError = 'invalidTransferAmount'
+    } else if (input === 'input'){
+      removedError = 'invalidDestAddress'
+    }
+
+    let tempArray = []
```

Comment:
same as my comment on the other PR. You dont need to use let if you're pushing items into an array, or adding object properties. const should be fine here.